### ISSUE TITLE
Height Normalization Queries

### ISSUE THREAD

**Issue by Archit-Sharma2024** [chico2121/bpe#25]
Hello,
Thank you for the awesome work on 2D Motion Similarity.
I had a query regarding the height normalization.
As given in the paper the skeleton size of sequence is reduced or increased by a scale factor b/w 0.5-1.5 (randomly)

Queries
1. Does the normalization happen for both training dataset and evaluation set using the codes provided?
2. Is annotation2motion function being used to normalize height in the motion sequences in the same range?
3. Can you please elaborate on the mean_height, how is it being calculated (is it for skeletons)?

Requesting your kind assistance.
Thank you
